hack/kv-smoke-tests.sh

```diff
@@ -250,6 +250,7 @@ ${TESTS_BINARY} \
     -oc-path="$(which oc)" \
     -kubectl-path="$(which oc)" \
     -utility-container-prefix=quay.io/kubevirt \
+    -utility-container-tag="${KUBEVIRT_VERSION}" \
```

Comment:
Aren't you missing `--container-tag=` too?

Reply:
I don't think so.

Reply:
I know for fact you need it at least for some storage tests
https://github.com/kubevirt/kubevirt/blob/0702cc5fc82deffa4534be8b4ee2b1ec7bf0f927/tests/storage/events.go#L119

Reply:
My concern is that it will be to wide change that will introduce non-existing images. This way I can limit it. I hope it's enough...

Reply:
ok. let's try.

Reply:
done